Extensions/probing.hh

```diff
@@ -324,6 +273,43 @@ inline double getReactivityScore(const Subsequence &inputSubseq,
   static double clusterPaired;
   std::string modifier = getProbing_modifier();
 
+  /* -store scores in a lookup matrix to avoid recalculations
```

Comment:
you might want to add to your comments, that the offset is only used in two-track modes